table/server/master/src/test/java/alluxio/master/table/UdbFilterSpecJsonSourceTest.java

```diff
@@ -0,0 +1,320 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.table;
+
+import static alluxio.master.table.UdbFilterSpecJsonSource.AbstractSpecObject.FIELD_TYPE;
+import static alluxio.master.table.UdbFilterSpecJsonSource.BypassTablesSpec;
+import static alluxio.master.table.UdbFilterSpecJsonSource.IncludeExcludeObject;
+import static alluxio.master.table.UdbFilterSpecJsonSource.NameObject;
+import static alluxio.master.table.UdbFilterSpecJsonSource.PartitionSpecObject;
+import static alluxio.master.table.UdbFilterSpecJsonSource.RegexObject;
+import static alluxio.master.table.UdbFilterSpecJsonSource.TablePartitionSpecObject.TYPE_PARTITION_SPEC;
+import static alluxio.master.table.UdbFilterSpecJsonSource.TablePartitionSpecObject.TYPE_NAME;
+import static alluxio.master.table.UdbFilterSpecJsonSource.TablePartitionSpecObject.TYPE_REGEX;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import alluxio.master.table.UdbFilterSpecJsonSource.NameOrRegexObject;
+import alluxio.master.table.UdbFilterSpecJsonSource.TablePartitionSpecObject;
+import alluxio.master.table.UdbFilterSpecJsonSource.TablePartitionSpecObject.Type;
+import alluxio.master.table.UdbFilterSpecJsonSource.TablesEntry;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException;
+import com.fasterxml.jackson.databind.exc.ValueInstantiationException;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class UdbFilterSpecJsonSourceTest {
+  private ObjectMapper mMapper;
+
+  @Before
+  public void before() {
+    mMapper = new ObjectMapper();
```

Comment:
same here , unnecessary `before`

Reply:
Fixed in 391b8b369413e0ba5c9528c2c0645cc54ee76932